src/main/java/com/iexec/sms/tee/session/scone/SconeSessionMakerService.java

```diff
@@ -67,7 +67,7 @@ public SconeSessionMakerService(
      */
     @NonNull
     public SconeSession generateSession(TeeSessionRequest request)
-            throws TeeSessionGenerationException {
+            throws TeeSessionGenerationException, IllegalArgumentException, IllegalStateException {
```

Comment:
`IllegalArgumentException` and `IllegalStateException` should no more be thrown here